app/views/review/list.html.haml

```diff
@@ -3,21 +3,19 @@
   %table
     %thead
       %tr
-        %th
-          Slug
-        %th
-          Notes
+        %th Title
+        %th Your Rating
+        %th Notes
         - @sortable_questions.each do |question|
           %th.sortable= question.label
-        %th
-          Updated
     %tbody
       - @proposals.each do |proposal|
+        - rating = proposal.ratings.where(user: current_user).first
         %tr
           %td
-            = link_to proposal.slug[0,6], review_path(@event, proposal), class: 'slug'
+            = link_to proposal.get_title, review_path(@event, proposal), class: 'slug'
+          %td.rating= "#{rating ? rating.score : 0}/5"
```

Comment:
Could we not default it to 0 to make it easier to spot the ones you haven't rated yet (just in case someone wants to give an real rating of 0)?